net/wgsd/files/wgsd-coredns.init

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+
+USE_PROCD=1
+NAME=wgsd-coredns
```

Comment:
This is not used anywhere.

Reply:
Do you mean `NAME`, because i saw other flags in other packages.

Reply:
Can you share with me some examples, please? I would like to check this out. Thanks!

Reply:
@BKPepe whole idea is described at author's post: https://www.jordanwhited.com/posts/wireguard-endpoint-discovery-nat-traversal/

So basically i have a VPS which works as a VPN exit node, i call it Cloud Router (CR, simply because i run openwrt x86 somewhere at cloud), and two sites behind NAT that wants to talk directly, as the one does video streaming.

On a CR i run wgsd-coredns daemon and have WG interface with both places peers configured (let's call them Alice and the Bob).
Then on a client side (Alice and Bob) i just have WG interface with the config to connect to the CR and Bob's/Alice's  peers. Then wgsd-client query the central node for the other peers IP/Port and update configuration of your WireGuard interface.

That works fine for SNAT, but seems something has changed on one of my ISPs. So i'm searching for other site-to-site mesh tunneling...